### PR TITLE
extension page updated

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: modules/install/pages/extensions.adoc [hunk 1]
```diff
@@ -1,24 +1,316 @@
...
+= Interpreting Extensions, Subsystems, and Profiles
 
-Brief blurb about the architecture of JBoss EAP (Do not repeat what will be already be covered in the Tech Overview course 1 in the path)
+* JBoss EAP 8.0 provides implementations of the Jakarta EE 10 APIs and Jakarta EE 10 https://jakarta.ee/specifications/webprofile/10/[Web Profile], https://jakarta.ee/specifications/coreprofile/10/[Core Profile], and the Full https://jakarta.ee/specifications/platform/10/[Platform] standards.
+* JBoss EAP 7.4, supported Jakarta EE 8. However, Jakarta EE 10 has undergone several changes, the most significant being the modification of the package namespace for Jakarta EE APIs from javax to jakarta.
+* See the complete list of specifications in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8.0/html-single/release_notes_for_red_hat_jboss_enterprise_application_platform_8.0/index#jakarta-ee-10-support_assembly-release-notes[release notes].
+
+== Modules in JBoss EAP
+
+* A module is a bundle composed of a library developed in Java and an XML configuration file that provides Jakarta Enterprise Edition (Jakarta EE) compliant functionalities.
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc]: Using bundle confuses students - A library is a bundle of classes. Give a clear definition of modules in straight forward simple language.

File: modules/install/pages/extensions.adoc [hunk 2]
```diff
@@ -1,24 +1,316 @@
...
+drwxr-xr-x.  4 jboss jboss   35 Apr  9 04:13 net
+drwxr-xr-x. 29 jboss jboss 4096 Apr  9 04:13 org
+drwxr-xr-x.  4 jboss jboss   34 Apr  9 04:13 sun
+drwxr-xr-x.  3 jboss jboss   17 Apr  9 04:13 wildflyee
+----
 
 == Extensions
 
-== Subsystems
+* Extensions are modules that extend the core capabilities of the server.
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc]: confusing. do you mean modules contain core functionality and extensions extend the feature set. simpler languages needed

File: modules/install/pages/extensions.adoc [hunk 3]
```diff
@@ -1,24 +1,316 @@
...
+    ...
+----
+
+* JBoss EAP uses the module name to find extension module. For example, to locate the `org.jboss.as.jpa` module, JBoss EAP searches for the `org/jboss/as/jpa/` directory in the `JBOSS_HOME/modules` directory.
+
+* The JBoss EAP core loads and activates modules/extensions only when necessary, so that inactive features do not consume memory or CPU resources unnecessarily.
+
+[NOTE]
+====
+If an organizations applications do not need a particular extension, then the system administrator can disable the extension so JBoss EAP does not load its management model anymore. Remove the <extension> entry and its corresponding <subsystem> section from the configuration file. This can be accomplished either by editing the XML directly, or using the Management CLI.
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc]: we are contradicting ourselves by saying edit XML directly. Remove this sentence. always use EAP CLI or web console to manage XML config

File: modules/install/pages/extensions.adoc [hunk 4]
```diff
@@ -1,24 +1,316 @@
...
+====
+
+== The Subsystems and Profiles
+
+* A subsystem is where you configure the extensions of the JBoss EAP instance.
+
+* A profile is a collection of subsystems.
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc:103]: MOve this sentence before the previous sentence

File: modules/install/pages/extensions.adoc [hunk 5]
```diff
@@ -1,24 +1,316 @@
...
+[NOTE]
+====
+Be careful not to confuse a JBoss EAP profile with a Jakarta EE profile. A JBoss EAP profile is a collection of subsystem configurations that are used to define the capabilities and services of a JBoss EAP server instance. A Jakarta EE profile is a collection of Java EE standards.
+====
+
+* In a managed domain, the administrator can create new profiles, either from scratch, or cloned from the ones provided, and then customize the new profiles before associating them to their respective groups.
+
+* A profile can also be a child of another profile, thus inheriting the subsystem configuration from its parent profile, so common configurations can be shared and maintained in a single place.
+
+== Lab 1: Starting different Subsystems
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc]: redudant and repeating. Remove this lab. Just include the steps to start one of the profiles as part of the lecture itself.

File: modules/install/pages/extensions.adoc [hunk 6]
```diff
@@ -1,24 +1,316 @@
...
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-full-ha.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+
+== Lab 2: Configuring Subsystems
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc]: keep this lab. remove lab 1. show how to start different profiles in lecture

File: modules/install/pages/extensions.adoc [hunk 7]
```diff
@@ -1,24 +1,316 @@
...
 
-* Unravel the actual file system layout and show some XML snippets and JBoss EAP CLI list commands
-* Some nice web console screenshots can go here...
-* Do not go too much into details about EAP CLI or web console here. Remember there is a separate chapter for EAP config...
-* Different modes - standalone, standalone-ha, domain etc,  but not too much detail. We have details in the advanced admin course
+. Select the CONSOLE row in the table. Click Edit. Change the Level from INFO to DEBUG. Then click Save.
++
+image::new.png[align="center"]
 
+* Stop the instance of the JBoss EAP listening on port 9990 by pressing `Ctrl+C` on the terminal where it is running and restart the instance to reflect the new changes. Observe the additional debug information provided by the server in the console that reflects the log level change made in the management console.
```
Comment (rsriniva) [inline modules/install/pages/extensions.adoc]: restart not required to see logging changes. most subsystems can be hot reloaded

**Review by rsriniva — CHANGES_REQUESTED** [review]
excellent work again. minor comments to be addressed before approval

File: modules/install/pages/extensions.adoc (hunk 1 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc]: reframed and simplified

File: modules/install/pages/extensions.adoc (hunk 2 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc]: reframed and simplified

File: modules/install/pages/extensions.adoc (hunk 3 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc]: Removed the note

File: modules/install/pages/extensions.adoc (hunk 4 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc:103]: moved

File: modules/install/pages/extensions.adoc (hunk 5 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc]: removed

File: modules/install/pages/extensions.adoc (hunk 6 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc]: removed and modified

File: modules/install/pages/extensions.adoc (hunk 7 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/extensions.adoc]: modified

**Review by rsriniva — APPROVED** [review]

**Pull request merged**